src/win/winapi.c

```diff
@@ -109,4 +114,12 @@ void uv_winapi_init() {
     pPowerRegisterSuspendResumeNotification = (sPowerRegisterSuspendResumeNotification)
       GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
   }
+
+  secur32_module = LoadLibraryA("secur32.dll");
```

Comment:
You're asking the wrong person I'm afraid 😛 I know absolutely nothing about the windows side, IIRC you wrote the windows part of this PR!